lib/util/interpolate.js

```diff
@@ -13,7 +13,12 @@ module.exports = (text, data) => {
     if (!data) {
         return text;
     }
-    return text.replace(/\{\{\s*([^{}]+?)\s*\}\}/g, (fullMatch, term) => {
+
+    // Substitution content for any {{ }} markers.
+    return text.replace(/\{\{([^{}]+?)\}\}/g, (fullMatch, term) => {
+
+        // Strip leading and trailing whitespace.
+        term = term.replace(/^\s+|\s+$/g, "");
```

Comment:
`term.trim()`?

Reply:
Haha! I knew there had to be a function for that.